corehq/apps/app_manager/tests/data/v2_diffs/yaml/commcare-profile-settings.yaml.diff.txt

```diff
@@ -0,0 +1,357 @@
+  - name: "Auto Update Frequency"
+    description: "How often CommCare mobile should attempt to check for a new, released application version."
+    id: "cc-autoup-freq"
+    values: ["freq-never", "freq-daily", "freq-weekly"]
+    value_names: ["Never", "Daily", "Weekly"]
+    default: "freq-never"
+    values_txt: "After login, the application will look at the profile's defined reference for the authoritative location of the newest version. This check will occur with some periodicity since the last successful check based on this property. freq-never disables the automatic check."
+    since: "1.3"
+    force: true
+  
+  - disabled: true
+    name: "Purge Frequency"
+    description: "How often the phone should attempt to purge any cached storage which may have expired"
+    id: "purge-freq"
+    values: ["0", "1", "7", "15", "31"]
+    value_names: ["Whenever Possible", "Daily", "Once a Week", "Twice a Month", "Once a Month"]
+    default: "0"
+    values_txt: "Any positive integer. Represents period of purging in days."
+    force: true
+  
+  - name: "Days for Review"
+    description: "Amount of time previously submitted forms remain accessible in the CommCare app."
+    id: "cc-days-form-retain"
+    values: ["1", "7",  "31", "92", "365", "-1"]
+    value_names: ["Just One Day", "One Week", "One Month", "Three Months", "One Year", "Forms are Never Removed"]
+    requires: "{features.sense}='false'"
+    default: "-1"
+    values_txt: "Any positive integer. Represents period of purging in days." 
+    since: "2.8"
+    force: true
+  
+  - name: "Logging Enabled"
+    description: "Whether logging of incidents should be activated on the client."
+    id: "logenabled"
+    values: ["Enabled", "Disabled"]
+    value_names: ["Enabled", "Disabled"]
+    default: "Enabled"
+    group: "log"
+    force: true
++ 
++ - name: "Advanced App Builder"
++   description: "Advanced App Builder"
++   id: "advanced_app_builder"
++   group: "sense"
++   values: ["true", "false"]
++   value_names: ["Yes", "No"]
++   default: "false"
++   widget: "bool"
```

Comment:
Doesn't look like a diff to me, or at least not the format I'm used to. It looks like the entire old file with `+` each line and the additions with `++` before each line, or something like that. It'd probably be more helpful and less error prone if the diff were a more compact format that only shows the differences.